### PR TITLE
[9] Add more test data

### PR DESCRIPTION
**Pull request by IDragonfire**

### CODE REVIEW

File: test-data.sql [hunk 1]
```diff
@@ -120,6 +120,31 @@ delete from avatars;
...
+insert into ban_revoke (ban_id, reason, author_id) values
+  (2, 'I want to show that you can revoke the ban, but keep the data', 1); 
+
+-- sample clans
+delete from clan_membership;
+delete from clan;
+insert into clan (id, name, tag, founder_id, leader_id, description) values
+  (1, 'Test Clan', '123', 1, 1, 'This is test clan is very cool'),
+  (2, 'No_UID Clan', '123', 4, 4, 'This is test clan is very cool'),
+  (3, 'Test Clan', '123', 2, 1, 'This is test clan is very cool');
```
Comment (micheljung) [inline test-data.sql]: Seeing this, I think we have to make `name` unique

Comment (IDragonfire) [inline test-data.sql]: oh yes, also the tag should be unique ^^

Comment (micheljung) [inline test-data.sql]: oh that too ^^

Comment (IDragonfire) [inline test-data.sql]: I was sure the stuff is also handled in the database, but at least the service handle it
https://github.com/FAForever/faf-java-api/blob/develop/src/main/java/com/faforever/api/clan/ClanService.java#L62
https://github.com/FAForever/faf-java-api/blob/develop/src/main/java/com/faforever/api/clan/ClanService.java#L65

But it should not be possible to create the clan over the json api ...
https://github.com/FAForever/faf-java-api/blob/develop/src/main/java/com/faforever/api/data/domain/Clan.java#L34

Comment (IDragonfire) [inline test-data.sql]: Issues created 
https://github.com/FAForever/faf-java-api/issues/108
https://github.com/FAForever/db/issues/122

**Comment by Brutus5000** [comment]
I rebased this PR and removed the no longer necessary `delete` statements (already at the top). Ready for merge.

**Pull request merged**